src/test_utils/es/es_test_config.js

```diff
@@ -0,0 +1,44 @@
+import { format as formatUrl } from 'url';
+import { resolve } from 'path';
+
+import { admin } from '../../../test/shield';
+
+const SECOND = 1000;
+const MINUTE = 60 * SECOND;
+
+export const esTestConfig = new class EsTestConfig {
+  getLibesvmStartTimeout() {
+    return process.env.TEST_ES_STARTUP_TIMEOUT || (5 * MINUTE);
+  }
+
+  getDirectoryForEsvm(uniqueSubDir) {
+    if (!uniqueSubDir) {
+      throw new Error('getDirectoryForEsvm() requires uniqueSubDir');
+    }
+
+    return resolve(__dirname, '../esvm', uniqueSubDir);
+  }
+
+  getBranch() {
+    return process.env.TEST_ES_BRANCH || 'master';
```

Comment:
Welp, turns out merging https://github.com/elastic/kibana/pull/13149 does just that 😃